.github/workflows/run_system_tests.yml

```
@@ -0,0 +1,21 @@
+name: Run system tests
+
+on:
+  workflow_call:
+  workflow_dispatch:
+jobs:
+  run_system_tests:
+    runs-on:
+      - self-hosted
+      - rdss-nidaqmxbot-${{ matrix.configuration }}
+    strategy:
+      matrix:
+        configuration: ["win-10-py32", "win-10-py64"]
+    timeout-minutes: 80
+    steps:
+      - name: checkout repository
```

consistency with other workflows
```suggestion
      - name: Check out repo
```